my_compassion/static/src/css/my2_login.css

```diff
@@ -0,0 +1,174 @@
+  height: 100px;
+  position: relative;
+  left: 15%;
+  transform: translateX(-50%);
+}
+
+/* Circular icon styling */
+.my2-login-box .icon {
+  height: 200px;
+  clip-path: ellipse(50% 50% at 50% 50%);
+}
+
+/* Hide logo on mobile devices */
+@media (max-width: 767px) {
+  .my2-login-box .logo {
+    display: none;
+  }
+}
+
+/* Welcome message styling */
+.my2-login-box .welcome {
+  font-weight: 600;
+  font-size: 1.25rem;
+  color: #005eb8;
+  margin-bottom: 1.5rem;
+}
+
+/* Form input field styling */
+.my2-login-form .form-control {
+  border-radius: 0.5rem;
+  padding: 0.75rem;
+  font-size: 0.95rem;
+}
+
+/* Form group spacing */
+.my2-login-form .form-group {
+  margin-bottom: 1rem;
+}
+
+/* Password toggle button styling */
+.toggle-password {
+  position: absolute;
+  right: 1rem;
+  top: 0;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  color: #999;
+}
+
+/* Primary button styling */
+.btn-primary {
+  background-color: #005eb8;
+  border-color: #005eb8;
+  border-radius: 0.5rem;
+  padding: 0.6rem 1rem;
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+/* Link styling within login box */
+.my2-login-box a {
+  text-decoration: underline;
+}
+
+/* Button hover effect */
+.btn-primary:hover {
+  background-color: #004a94;
+}
+
+/* Forgot password link styling */
+.forgot-password {
+  font-size: 1rem;
+  color: #005eb8;
+  text-decoration: none;
+  text-align: left !important;
+  margin-left: 0;
+}
+
+/* Forgot password hover effect */
+.forgot-password:hover {
+  text-decoration: underline;
+}
+
+/* Mobile responsiveness for login box */
+@media (max-width: 768px) {
+  .my2-login-box {
+    max-width: 100%;
+  }
+}
+
+/* Language picker container styling */
+.language-picker {
+  padding-top: 20rem;
+  min-height: calc(100vh - 300px);
+  position: absolute;
+  right: 15%;
+  transform: translateX(50%);
+}
+
+/* Language picker wrapper styling */
+.language-picker-wrapper {
+  position: relative;
+  display: inline-block;
+}
+
+/* Globe icon before language picker */
+.language-picker-wrapper::before {
+  content: "\f0ac";
+  font-family: "Font Awesome 5 Free";
+  font-weight: 900;
+  position: absolute;
+  left: 10px;
+  top: 50%;
+  transform: translateY(-50%);
+  color: #005eb8;
+  z-index: 2;
+  pointer-events: none;
+}
+
+/* Language picker select element styling */
+#language-picker {
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  padding: 8px 30px 8px 35px;
+  font-size: 14px;
+  color: #005eb8;
+  border: 1px solid #005eb8;
+  border-radius: 20px;
+  background-color: white;
+  background-image: url("data:image/svg+xml,%3Csvg fill='blue' height='10' viewBox='0 0 20 20' width='10' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M5.293 7.293L10 12l4.707-4.707-1.414-1.414L10 9.172 6.707 5.879 5.293 7.293z'/%3E%3C/svg%3E");
+  background-repeat: no-repeat;
+  background-position: right 10px center;
+  background-size: 10px;
+  width: auto;
+  min-width: 120px;
+}
```

Comment:
This seems related to the menu, it should be place else where then my2_login.css and be part of another PR
```suggestion
```